package.json

```diff
@@ -54,7 +54,7 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.2",
     "babel-loader": "^8.0.4",
-    "babel-plugin-styled-components": "^1.10.1",
+    "babel-plugin-styled-components": "1.10.5",
```

Comment:
Just wondering - why are we pinning to a specific version here, instead of having the caret for matching newer versions like this `"^1.10.5",`?

Reply:
Discussed offline